{"name": "Demarrage", "status": "failed", "statusDetails": {"message": "Setup failed:\nMaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000028D32E32510>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))\n\nAlso teardown failed:\nNo application is open"}, "start": 1689267371095, "stop": 1689267382082, "uuid": "cd687d83-4ae0-4a0e-bd73-3e04afe1240f", "historyId": "84a88efa2949dbb22117d72f5b94008a", "fullName": "Tests.homeTest.Demarrage", "labels": [{"name": "suite", "value": "Tests"}, {"name": "subSuite", "value": "homeTest"}, {"name": "framework", "value": "robotframework"}, {"name": "language", "value": "cpython3"}, {"name": "host", "value": "LAPTOP-KC6KER4Q"}, {"name": "thread", "value": "default-471b3ca2-938f-492a-b412-ec9f1e936cad"}, {"name": "tag", "value": "test_001"}]}